LabExT/View/Movement/LoadStoredCalibrationWindow.py

```diff
@@ -61,6 +57,22 @@ def __init__(
         with open(self.mover.calibration_settings_file, "r") as fp:
             self.calibration_settings = json.load(fp)
 
+        if self.calibration_settings.get("chip_name") != self.chip.name:
+            raise RuntimeError(
+                f"There is no stored calibrarion for the chip name '{self.chip.name}'."
+                f"Instead found one for chip name '{self.calibration_settings.get('chip_name')}'")
```

Comment:
I would only raise a warning here and not fail. Could be that the user did a typo? Force-failing on unequal chip names is a bit patronizing.

Reply:
Changed :)